pymess/models/common.py

```diff
@@ -95,14 +110,22 @@ def clean_body(self, context_data=None):
     def get_body(self):
         return self.body
 
-    def can_send(self, recipient, context_data):
-        return self.is_active
+    def can_send_for_object(self, related_objects):
+        if hasattr(self, 'disallowed_objects'):
+            return (
+                not self.disallowed_objects.filter_from_related_objects(related_objects).exists()
+            ) if related_objects is not None else True
+        else:
+            return True
```

Comment:
if else pattern?

`
return not hasattr(self, 'disallowed_objects') or related_objects is None or  not self.disallowed_objects.filter_from_related_objects(related_objects).exists()
`

Reply:
bývalo to v průběhu vývoje o dost složitější, díky

Reply:
please change  the code